test/Romina.Api.UnitTests/Controllers/ProductControllerTest.cs

```diff
@@ -84,9 +63,9 @@ public void GetId_WhenProductIdDoesNotExist_ThenReturnNotFound()
         public void GetByQuery_WhenProductsExist_ReturnProducts()
         {
             List<Product> listOfProducts = new List<Product>();
-            listOfProducts.Add(_nikeGlove);
-            listOfProducts.Add(_nikeTrouser);
-            listOfProducts.Add(_nikeHat);
+            listOfProducts.Add(It.IsAny<Product>());
+            listOfProducts.Add(It.IsAny<Product>());
+            listOfProducts.Add(It.IsAny<Product>());
```

Comment:
hey bud, its important to use methods like `It.IsAny<>` at the right place. They should only be used when **setting** up mocks. 

For example, here's it being used correctly `_productHandler.DoSomething(It.IsAny<string>).Returns("something cool")`. Here we are saying that when the method `DoSomething` is invoked and `any` string is provided we want to return the string `"something cool"`. 

In your case, you are creating a list of products to return (in the arrange part of your test). Ideally we want this `listOfProducts` object to contain data that we can later assert on. So even though `listOfProducts.Add(It.IsAny<Product>())` compiles, its not quite right. 

In this test, we don't actually assert on the contents of `listOfProducts`, we just assert the list is not null on line 76, so you can actually **remove** lines 66-68. 

If this don't make sense lemme know and we can run through it together